Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesInterface.h

```diff
@@ -314,6 +320,8 @@ namespace AzFramework
         virtual void ClaimEntities(
             EntitySpawnTicket& ticket, ClaimEntitiesCallback listCallback, ClaimEntitiesOptionalArgs optionalArgs = {}) = 0;
 
+        virtual void ClaimEntity(AZ::EntityId entityId, void* ticket, ClaimEntityOptionalArgs optionalArgs = {}) = 0;
```

Comment:
This is using a void* ticket because when the entity gets added to the game context, it only has access to void* and not the EntitySpawnTicket. 
If we use a map of <TicketId, Ticket*> like suggested in the other comment, then this can use EntitySpawnTicket& like the others. Will do that once a decision is reached on the other thread on whether we should use lookup tables or not.

Reply:
That makes the `ClaimEntity` function only useful for that one use case and anybody using the interface properly with a ticket won't be able to call this. Imho this violates the contract of the interface and shouldn't be done this way.

Reply:
Removed the claimEntity() call and made sure that all new interface calls use EntitySpawnTicket& like the rest of the calls in the interface.